src/request_handler.rb

```diff
@@ -0,0 +1,66 @@
+require 'yaml'
+
+class RequestHandler
+  attr_reader :message, :bot, :receiver
+
+  def initialize(receiver)
+    @receiver = receiver
+    @message = receiver.message
+    @bot = receiver.bot
+
+    @answers = YAML.load_file(File.dirname(__FILE__) + '/data/messages.yml')
+
+    handle_message
+  end
+
+  def handle_message
+    case message.text
+      when 'What time is it?'
+        send_response Time.now.strftime("%T")
+      when '/start'
+        send_response @answers.fetch('start')
+        send_response @answers.fetch('commands')
+      when '/stop'
+        send_response @answers.fetch('stop')
+      when '/translate'
+        send_response @answers.fetch('translate')
+
+        receiver.listen do |message|
```

Comment:
I would rather move these blocks outside to separate methods, but it's not a big deal.